src/app/combobox/example/combobox-example.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="padding-15">
+  <div class="row">
+    <div class="col-sm-12">
+      <h4>Combobox Component Example</h4>
+      <hr/>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <div class="form-group">
+        <pfng-combobox [items]="states" [disabled]="disableComponent"></pfng-combobox>
+      </div>
+    </div>
+  </div>
+  <div class="row padding-top-10">
+    <div class="col-sm-12">
+      <h4 class="actions-label">Settings</h4>
+      <hr/>
+    </div>
+  </div>
+  <div class="row">
```

Comment:
It would be good to add an actions text area here to capture any events emitted (e.g., when a selection has been made.)